angularjs-portal-home/src/main/webapp/my-app/layout/widget/partials/widget-card.html

```diff
@@ -84,26 +84,30 @@
       </div>
 
       <!-- For basic apps (not a widget, not simple content, no pithy content), show only an icon -->
-      <a tabindex="-1" ng-switch-when="NORMAL" ng-href="{{widgetCtrl.renderURL(portlet)}}" target="{{portlet.target}}" class="normal-widget">
-        <div class="widget-icon-container" layout="column" layout-align="center center">
-          <portlet-icon></portlet-icon>
-        </div>
+      <div ng-switch-when="NORMAL">
+        <a tabindex="-1" ng-href="{{widgetCtrl.renderURL(portlet)}}" target="{{portlet.target}}" class="normal-widget">
```

Comment:
Interesting...  Yeah, probably don't want screen readers or key navigation to hit something that's redundant.  At some point, we made the launch button optional.  I don't think that was an option for these normal widgets?